core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -6217,7 +6217,7 @@ LogicalProject(EXPR$0=[1])
       LogicalProject(DEPTNO=[$7])
         LogicalFilter(condition=[>($7, 1)])
           LogicalTableScan(table=[[CATALOG, SALES, EMP]])
-    LogicalFilter(condition=[OR(>($7, 7), >($7, 1))])
+    LogicalFilter(condition=[SEARCH($7, Sarg[(1..+∞)])])
       LogicalTableScan(table=[[CATALOG, SALES, EMP]])
```

Comment:
An improvement change.

Reply:
It should have been `>($7, 1)`

Reply:
What's the difference ?

Reply:
`>($7, 1)` is easier to follow than `SEARCH($7, Sarg[(1..+∞)])`.

Reply:
I agree, but there are already many `SEARCH($7, Sarg[(1..+∞)])` nodes in our plans, we can fire another issue to promote it but not in this patch.

Reply:
I have fixed it, thanks.